### ISSUE TITLE
Give moderator access to the last 5 minutes or so log or connection log

### ISSUE THREAD

**Issue by Chairn** [ddnet/ddnet#5433]
This should ease moderation when someone enters a server spam some racist shit and leaves instantly. 

We just had a case tonight and @bencie had to chase him around until he finally get his IP. Then the guy changed IP, then changed name...

**Comment by Learath2** [comment]
Should we maybe keep track of ips and names for a reasonable duration? Maybe 48 hours or so atleast? Such a mapping could be used by mods to check the ip of someone who already left

**Comment by Chairn** [comment]
48 hours seems overkill. 5 minutes leaves enough time for a moderator to connect. Maybe 1 or 2 hours max. Maybe a moderator could give his opinion?

**Comment by bencie** [comment]
I love the idea, but I think this should be command from status, as this could clutter up the console alot. 5 minutes seems reasonable, I can't name an instance where I would've needed a person's ip who has already left hours ago. More time wouldn't hurt though.

**Comment by Vy0x2** [comment]
I like the idea, but what should be logged? Disconnects? Chat messages? I think they are the most important ones and filtering these would result in less spam using this "last 5 min log" command
Btw 5 min is enough imo

**Comment by def-** [comment]
Duplicate of https://github.com/ddnet/ddnet/issues/5433

**Comment by Chairn** [comment]
Duplicate of itself? You meant https://github.com/ddnet/ddnet/issues/5796 i think :)

**Comment by Vy0x2** [comment]
Why, those are two different suggestions

This recommends to just show the last 5 minutes of logs (way easier to implement imo)
and the other issue wants to create a database of banned players and store their ips

**Comment by Chairn** [comment]
But the idea is similar. Both starts from the fact that moderators don't know other moderator's actions.

**Comment by Vy0x2** [comment]
I disagree, this issue description is about players who join, spam and leave instantly.
The issue or feature request #5796 does not address the same problem as this issue.

**Comment by heinrich5991** [comment]
Reopening as per request of @Vy0x2.
